cfn.yaml

```diff
@@ -49,10 +51,144 @@ Resources:
               Action:
                 - lambda:InvokeFunction
```

Comment:
Do you need this if we're integrating via SQS?

Reply:
I think so, particularly as the EventMapper magically inherit the role and therefore needs to have the right to trigger the lambda itself